bazel/check_deps/BUILD

```diff
@@ -13,10 +13,9 @@ load("@rules_python//python:defs.bzl", "py_test")
 filegroup(
     name = "non_test_cc_rules",
     data = [
-        "//toolchain/install:carbon-busybox",
-        # The tree sitter rules can't be queried; evaluation fails on
-        # @platforms.
-        # "//utils/tree_sitter",
+        "//bazel/malloc:tcmalloc_if_linux_opt",
+        "//toolchain/install:carbon_toolchain_tar_gz_rule",
```

Comment:
Worth skipping the `gz` version? My memory was that was much slower to build.

Reply:
Hmm, or maybe this doesn't matter because we're just going to query over this filegroup, not actually build it?